src/views/UserPanelView/Components/UsersTable.jsx

```diff
@@ -15,30 +15,16 @@ import LoadingProgress from './../../../components/Progress'
 import Context from '../../../context/appContext'
 import useUsers from './../../../hooks/useUsers'
 
-export function UsersPanelTable ({ classes }) {
-  const { users, isLoading, page, handleChangePage, rowsPerPage, setLocation } = useUsers()
+export function UsersPanelTable ({ classes, roleValue }) {
+  const { VITE_ENV: env } = import.meta.env
+  const groupRole = `${env}_${roleValue}_group`
+  const { users, isLoading, page, handleChangePage, rowsPerPage, setLocation } = useUsers({ groupRole })
   const { setSelectedEmail } = useContext(Context).user
 
   const changeRoute = () => {
     setLocation('/admin/users/detail/')
   }
 
-  const getRoles = (groups) => {
-    const roles = groups || []
-    return (
-      <div key={`roles-${groups}`} style={{ display: 'flex' }}>
-        {roles.map((role) => (
-          <Chip
-            key={role}
-            label={role}
-            variant="outlined"
-            className={classes.roleName}
-          />
-        ))}
-      </div>
-    )
-  }
-
   return (
     <div className={classes.root}>
```

Comment:
If we are using mui components we should use Box or fragment <> </>, in this case you can use the box component